alacritty/src/event.rs

```diff
@@ -436,16 +436,18 @@ impl<N: Notify> Processor<N> {
                         processor.on_mouse_wheel(delta, phase, modifiers);
                     },
                     Focused(is_focused) => {
-                        processor.ctx.terminal.is_focused = is_focused;
-                        processor.ctx.terminal.dirty = true;
+                        if window_id == processor.ctx.window.window_id() {
```

Comment:
We can use a match guard here. That should allow us to just reduce level of indentation by 1 for basically free.